src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Truck start work");
+    }
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stop work");

```

Comment:
The message in the `stopWork` method should be more informative and indicate the specific type of machine that has stopped working. For example: 'Truck has stopped working.'